api/permissions_api.py

```diff
@@ -25,22 +25,37 @@ class PermissionsAPI(basehandlers.APIHandler):
 
   def do_get(self, **kwargs):
     """Return the permissions and the email of the user."""
-
     # No user data if not signed in
     user_data = None
 
     # get user permission data if signed in
     user = self.get_current_user()
     if user:
-      user_data = {
-        'can_create_feature': permissions.can_create_feature(user),
-        'approvable_gate_types': sorted(
-            approval_defs.fields_approvable_by(user)),
-        'can_comment': permissions.can_comment(user),
-        'can_edit_all': permissions.can_edit_any_feature(user),
-        'is_admin': permissions.can_admin_site(user),
-        'email': user.email(),
-        'editable_features': permissions.feature_edit_list(user)
-      }
+      user_data = self.get_all_perms(user)
+      paired_user = self.find_paired_user(user)
+      if paired_user:
+        user_data['paired_user'] = self.get_all_perms(paired_user)
 
     return {'user': user_data}
+
+  def get_all_perms(self, user):
+    """Return a dict of permissions for the given user."""
+    return {
+      'can_create_feature': permissions.can_create_feature(user),
+      'approvable_gate_types': sorted(
+          approval_defs.fields_approvable_by(user)),
+      'can_comment': permissions.can_comment(user),
+      'can_edit_all': permissions.can_edit_any_feature(user),
+      'is_admin': permissions.can_admin_site(user),
+      'email': user.email(),
+      'editable_features': permissions.feature_edit_list(user)
+    }
+
+  def find_paired_user(self, user):
+    """If @google.com or @chromium.org, return the other one."""
+    username, domain = user.email().split('@', 1)
+    if domain == 'google.com':
+      return User(email=username + '@chromium.org')
```

Comment:
Would it be feasible/useful to check if a matching user exists with a @chromium.org account if the user is using any non-chromium.org email? Ex. @gmail.com or @microsoft.com? Just curious

Reply:
It's possible.  We could add that if users request it.  However, I don't think that very many chrome contributors mistakenly use their @gmail.com accounts on chromestatus.  I think that people are aware of which accounts they are using and know the difference between work and personal accounts, but the chromium and google accounts are both work accounts with similar purposes.